_data/openseeds/ols-9/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+# List of experts, possible mentors and organizers for openseeds ols-9
+#
+#
+# People should be also in people.yaml file and linked using their GitHub username
+# Ordering by expertise should be done by running the bin/sort-expertises.py script
+---
+experts: []
+organizers:
+- bebatut
+- emmyft
```

Comment:
```suggestion
```

Reply:
Who should be added from the team?

Reply:
@malvikasharan, @yochannah

Reply:
@bebatut - Maybe asking them in person will help. :-)